acme/acme/fields.py

```diff
@@ -57,7 +57,7 @@ class Resource(jose.Field):
     def __init__(self, resource_type: str, *args: Any, **kwargs: Any) -> None:
         self.resource_type = resource_type
         super().__init__(
-            'resource', default=resource_type, *args, **kwargs)
+            'resource', default=resource_type, *args, **kwargs)  # type: ignore[misc]
```

Comment:
What do you think about:
```
            kwargs['default'] = resource_type
            super().__init__('resource', *args, **kwargs)
```
This seems to make mypy happy.

Reply:
Indeed, way better !